Open duck mini v2

apirrone opened the pull request:
Adds mjcf description for [Open Duck Mini v2](https://github.com/apirrone/Open_Duck_Mini)

kevinzakka commented on open_duck_mini_v2/README.md:
```diff
@@ -0,0 +1,19 @@
+# Open Duck Mini V2
```
Hi @apirrone, can you enhance this README to match other models? (e.g. [here](https://github.com/google-deepmind/mujoco_menagerie/blob/main/arx_l5/README.md)). Broadly we need to know where the source of truth was derived from (if from a URDF, a commit SHA + link to the URDF would be great) and what steps were taken to create the MJCF.

kevinzakka commented on open_duck_mini_v2/open_duck_mini_v2.xml:
```diff
@@ -0,0 +1,454 @@
+<mujoco model="onshape">
```
Can you do some minimal formatting on this onshape generated XML? We have some instructions [here](https://github.com/google-deepmind/mujoco_menagerie/blob/main/CONTRIBUTING.md#xml-style).

kevinzakka commented on open_duck_mini_v2/open_duck_mini_v2.xml:
```diff
@@ -0,0 +1,454 @@
+  </contact>
+  <worldbody>
+    <body name="base" pos="0 0 0.22">
+      <!-- <body name="base" pos="0 0 0.4"> -->
```
Nit: Remove comments please.

kevinzakka commented on open_duck_mini_v2/open_duck_mini_v2.xml:
```diff
@@ -0,0 +1,454 @@
+                  rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_right_sheet"/>
+                <geom pos="0.01606 0.14365 0.10915" quat="0.707107 -0.707107 0 0" type="mesh"
+                  rgba="0.223529 0.219608 0.219608 1" mesh="left_knee_to_ankle_left_sheet"/>
+                <geom pos="0 -0.05315 0.00955" quat="0.5 -0.5 -0.5 0.5" type="mesh" rgba="0.247059 0.243137 0.243137 1"
```
I'd prefer if `rgba`s were converted to materials and material names used instead.

kevinzakka reviewed (changes requested):
Thanks for your contribution @apirrone! I left some comments.

apirrone commented:
Hi @kevinzakka, thanks for the comments ! 

I actually have a question : The mjcf I added here is a heavily decimated and simplified version intended to be efficient for mjx, but I can also export a pretty one with all the geometry and features. 

Which one makes most sense to publish here? Can I upload both ? 

Thanks !

kevinzakka commented:
Yes making both is fine. Note visual geometry should not affect MJX performance if you are using primitive geoms for the colliders.
